backend/api/Startup.cs

```diff
@@ -106,6 +106,7 @@ public void ConfigureServices(IServiceCollection services)
             });
             services.Configure<JsonSerializerOptions>(options =>
             {
+                options.ReferenceHandler = ReferenceHandler.IgnoreCycles;
```

Comment:
This is new with dotnet6. It allows the json to work even with recurcive structures by ignoring them.